src/tooltip/tooltip.js

```diff
@@ -260,6 +260,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
              * Observe the relevant attributes.
              */
             attrs.$observe( type, function ( val ) {
+              if (val && val.length >= 1 && val.charAt(0) == '@') {
```

Comment:
The error is in checking for `=> ` as it should rather be `>`

Thanks for catching this